soc/xtensa/intel_adsp/ace/include/intel_ace15_mtpm/adsp_watchdog.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * Author: Adrian Warecki <adrian.warecki@intel.com>
+ */
+
+#ifndef ZEPHYR_SOC_INTEL_ADSP_WATCHDOG_H_
+#define ZEPHYR_SOC_INTEL_ADSP_WATCHDOG_H_
+
+/**
+ * @brief Pause watchdog operation
+ *
+ * Sets the pause signal to stop the watchdog timing
+ *
+ * @param dev Pointer to the device structure
+ * @param channel_id Channel identifier
+ */
+int intel_adsp_watchdog_pause(const struct device *dev, const int channel_id);
```

Comment:
why the custom watchdog calls? Shouldnt we extend the existing zephyr API and add those instead?

Reply:
@nashif It is the only driver that has this option. Should I extend the watchdog api for it?

Reply:
yes, this will be the preference, but this can be done in a followup PR